news/2023-01-01-osumonthly-6.md

```diff
@@ -0,0 +1,331 @@
+
+For the most voted play of the year, it was decided that [applerss](https://osu.ppy.sh/users/983349/taiko) insane FC of [Scattered Faith](https://osu.ppy.sh/scores/taiko/175098621). I mean, what else is there to say about this score? Scattered Faith is infamous in the osu!taiko community for being one of, if not, the fastest map currently ranked. Coming in at over 8 minutes long, and reaching up to 360bpm, this play is definitely deserving of its #1 finish.
+
+You can view all of the plays listed in the playlist below:
+
+<iframe src="https://www.youtube-nocookie.com/embed/videoseries?list=PLbY6YsbyBvHBnVWWAfpMe31kx6b95GaAa" allow="autoplay; encrypted-media" allowfullscreen="" width="100%" height="315" frameborder="0"></iframe>
+
+### Notable osu!catch plays of the year
+
+Starting off the list, we find [Motion](https://osu.ppy.sh/users/3885626)'s play on the 1.1x speed difficulty of [Hug and Kill](https://osu.ppy.sh/scores/fruits/210787391), a map infamous for the insane speed sections that very few players can even pass. Nevertheless, Motion shreds through all the speed section and almost clutches the FC, however even with his 2 misses he beat the previous best misscount by 20.
+
+Next up we have Motion's 4 miss on the highest difficulty of [XHRONOXAPSULE](https://osu.ppy.sh/scores/fruits/210576459), a map that currently sits as the ranked map with the highest star rating in the game, coming in at 10.07 stars. The scariest and toughest part of this map is definitely the end section which features 277bpm antiflow patterns, what makes this play so "special" is that he actually hit the hardest part of the map, a section that no other player had hit before, but failed at a previous, easier section, turning this play into a reverse choke. However, this isn't the play that sits in the leaderboard, since shortly after Motion would set yet another high score with better combo but 3 more misses, all located at the end of the map.
+
+The [Shin dan](https://osu.ppy.sh/beatmapsets/884597#fruits/1872971) might just be the hardest map compilation in osu!catch history. After all, surviving through a 10.43 star map that lasts for almost 10 minutes doesn't seem like an easy feat, however, after a great community effort, [ArMa79](https://osu.ppy.sh/users/4982799/fruits) claimed the first clear on this map, which in this context means, obtaining a 99% on the map. This isn't the end of the story though, as only a couple months later, Motion too would rise up to the challenge, sniping ArMa79's score in the process and setting one of the best plays of 2022.
```

Comment:
no link to the play?

Reply:
https://github.com/ppy/osu-wiki/pull/8570/commits/51385943ddb319ef202a6c8ca03aa2b2ff6467a2